runtime-modules/membership/src/lib.rs

```diff
@@ -198,17 +192,11 @@ pub struct InviteMembershipParameters<AccountId, MemberId> {
     /// New member controller account.
     pub controller_account: AccountId,
 
-    /// New member user name.
-    pub name: Option<Vec<u8>>,
-
     /// New member handle.
     pub handle: Option<Vec<u8>>,
 
-    /// New member avatar URI.
-    pub avatar_uri: Option<Vec<u8>>,
-
-    /// New member 'about' text.
-    pub about: Option<Vec<u8>>,
+    /// Meta data concerning new member.
```

Comment:
Please, make an extended comment on what is the possible content of the field (name, family name, etc).